app/src/main/java/emu/skyline/input/onscreen/OnScreenControllerView.kt

```diff
@@ -272,6 +273,14 @@ class OnScreenControllerView @JvmOverloads constructor(context : Context, attrs
         invalidate()
     }
 
+    fun getTextColor() : Int {
+        return controls.globalTextColor
+    }
+
+    fun getBGColor() : Int {
```

Comment:
`getBackgroundColor`, don't shorten to BG. Swap out BG for background also for all other functions in this and in other files.

Reply:
since setBackground is already used by on of the parents i was going with BGColor, i've setted it to BackGround color instead